### PR TITLE
netmap: migrate from the old storage format

### PR DESCRIPTION
**Pull request by fyrchik**
Need this for a release.
Tested on neofs-adm from https://github.com/nspcc-dev/neofs-node/pull/1903

Signed-off-by: Evgenii Stratonikov <evgeniy@morphbits.ru>

### CODE REVIEW

**Review by carpawell — DISMISSED** [review]

File: netmap/netmap_contract.go [hunk 1]
```diff
@@ -94,6 +94,24 @@ func _deploy(data interface{}, isUpdate bool) {
 
 	if isUpdate {
 		common.CheckVersion(args.version)
+
+		prefix := []byte(snapshotKeyPrefix)
+		for i := 0; i < DefaultSnapshotCount; i++ {
```
Comment (cthulhu-rider) [inline netmap/netmap_contract.go:99]: Shouldn't we read snapshot count from `snapshotCountKey`?

File: netmap/netmap_contract.go [hunk 2]
```diff
@@ -94,6 +94,24 @@ func _deploy(data interface{}, isUpdate bool) {
...
+			key := append(prefix, byte(i))
+			data := storage.Get(ctx, key)
+			if data != nil {
+				nodes := std.Deserialize(data.([]byte)).([]Node)
```
Comment (cthulhu-rider) [inline netmap/netmap_contract.go:103]: Can we decode to list of structures with old format in order to not loose the state?

Comment (fyrchik) [reply netmap/netmap_contract.go:103]: I think it is ok now -- the only new state is maintenance and it is added in this release, so all ready stored nodes are `Online`.

File: netmap/netmap_contract.go (hunk 1 above)
Comment (fyrchik) [reply netmap/netmap_contract.go]: Fixed

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**